pages/common/git-rebase.md

```diff
@@ -1,23 +1,24 @@
 # git rebase
 
 > Apply local commits on top of another branch's history.
+> This effectively "moves" an entire branch to another base, by creating copies of the commits in the new location.
 
-- Rebase your local branch interactively with the latest changes in local master:
+- Rebase the current branch on top of the master branch:
 
-`git rebase -i master`
+`git rebase master`
 
-- Rebase your local branch interactively with the latest changes from upstream:
+- Start an interactive rebase, which allows the commits to be reordered, omitted, combined or modified:
 
-`git fetch origin; git rebase -i origin/master`
+`git rebase -i {{target_base_branch}}`
 
-- Handle an active rebase merge failure, after editing conflicting file(s):
+- Continue a rebase that was interrupted by a merge failure, after editing conflicting files:
 
 `git rebase --continue`
 
-- Abort a rebase in-progress:
+- Abort a rebase in progress (e.g. if it is interrupted by a merge conflict):
 
 `git rebase --abort`
 
-- Rebase your local branch by specifying new base commit and old base commit:
+- Rebase a branch starting from a specific base commit, rather than the common ancestor shared with the target branch:
 
 `git rebase --onto {{new_base_commit}} {{old_base_commit}}`
```

Comment:
IMHO taking explanation in line 22 as context, it would be clearer if 

  * `{{new_base_commit}}` and `{{old_base_commit}}` are renamed using words appeared in line 22

      * e.g. `git rebase --onto {{target_commit}} {{starting_base_commit}}`

NOTE: `{{target_commit}}` is a wider usage and of course it can be specified with some target branch